File: 2798.number-of-employees-who-met-the-target.java
/*
 * @lc app=leetcode id=2798 lang=java
 *
 * [2798] Number of Employees Who Met the Target
 */

// @lc code=start
class Solution {
    public int numberOfEmployeesWhoMetTarget(int[] hours, int target) {
        
        int targetMet = 0;

        for(int i = 0; i < hours.length; i++){
            if(hours[i] >= target) targetMet++;
        }

        return targetMet;
    }
}
// @lc code=end

File: 2798.number-of-employees-who-met-the-target.js
/*
 * @lc app=leetcode id=2798 lang=javascript
 *
 * [2798] Number of Employees Who Met the Target
 */

// @lc code=start
/**
 * @param {number[]} hours
 * @param {number} target
 * @return {number}
 */
var numberOfEmployeesWhoMetTarget = function(hours, target) {
    
    let targetMet = 0;

    hours.forEach((hour)=>{
        if(hour >= target) targetMet++;
    })

    return targetMet;
};
// @lc code=end

